recipes/python-utilities-jsm/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
This license is already packaged so the file can be removed here. However, this license is not so it needs to be manually added and it would be good if you specify which one is correct: https://github.com/jonathanmorgan/python_utilities/blob/master/LICENSE.GPL

Reply:
I removed the referenced license file from the python_utilities repository and built a new version of the package with it removed (1.1.2), the LGPL is the correct license. I updated the recipe to reference the new version of python_utilities_jsm. I'll accept these changes. Is there anything else I need to do? Thanks for your help, and sorry for the delay.

Reply:
I also removed the LICENSE file from this recipe, since it is already packaged.